example/config/experiment.yaml

```diff
@@ -336,6 +356,14 @@ scoring:
     training_metric_groups:
         -
             metrics: [accuracy]
+    subsets:
+        -
+            name: women
+            query: |
+                select distinct entity_id
+                from demographics p
```

Comment:
I'm guessing this aliasing is leftover from when this query looked at predictions